docker-compose.yml

```
@@ -0,0 +1,21 @@
+version: "3.9"
+
+services:
+  production:
+    image: todo-app:prod
+    env_file:
+      - env.list
+    ports:
+      - "80:80"
+
+  development:
+    image: todo-app:dev
```

This relies on the user having already built this image. Rather than image, I tend to encourage using `build` so that the image is built for the user and we can remove another step:
```suggestion
    build:
      target: development
      context: .
```

(And similarly for the production container)